app/uk/gov/hmrc/securemessage/controllers/binders/package.scala

```diff
@@ -37,6 +38,19 @@ package object binders {
         s"${customerEnrolment.key}~${customerEnrolment.name}~${customerEnrolment.value}"
     }
 
+  implicit def queryStringBindableLanguage(
+    implicit stringBinder: QueryStringBindable[String]): QueryStringBindable[Language] =
+    new QueryStringBindable[Language] {
+
+      override def bind(key: String, params: Map[String, Seq[String]]): Option[Either[String, Language]] =
+        stringBinder.bind("lan", params) map {
```

Comment:
Instead of **lan**, the bind key will be **language** as given in the app.routes?

Reply:
Maybe worth adding IT tests to prove the routes binding works as expected, unit tests would cover only the binders and not the routes ?

Reply:
@kjainsc binders will take care of that. Since it is optional we dont need to worry for that now. Had unit tests anyway..